sessionspaces/src/permissionables/ldap.rs

```diff
@@ -0,0 +1,42 @@
+use crate::SessionSpaces;
+use ldap3::{LdapConnAsync, Scope, SearchEntry};
+
+pub async fn update_gid(sessions: &mut SessionSpaces) {
```

Comment:
It would likely be neater to have a method which fetched all the posix attributes of groups - like `fetch` in the other `permissionables`  - and combine these downstream. Means that we can later drop the ISPyB ones when move stuff is in ldap

Reply:
makes sense